documentstore_migracao/export/sps_package.py

```diff
@@ -103,6 +103,42 @@ def publisher_id(self):
         except IndexError:
             return None
 
+    @publisher_id.setter
+    def publisher_id(self, value):
+        if self.publisher_id is None:
+            pid_node = etree.Element("article-id")
+            pid_node.set("pub-id-type", "publisher-id")
+            pid_node.text = value
+            self.article_meta.insert(0, pid_node)
+        else:
+            pid_node = self.article_meta.xpath(
```

Comment:
@patymori esta instrução não pode causar: `IndexError: list index out of range`?